docs/source/testing.md

```diff
@@ -0,0 +1,76 @@
+# Tests
```

Comment:
Let's stick with either "Testing" or "Tests" in  both filename and section title.